package.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<package version="2.0" 
+	xmlns="http://pear.php.net/dtd/package-2.0" 
+	xmlns:tasks="http://pear.php.net/dtd/tasks-1.0" 
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://pear.php.net/dtd/tasks-1.0
+http://pear.php.net/dtd/tasks-1.0.xsd
+http://pear.php.net/dtd/package-2.0
+http://pear.php.net/dtd/package-2.0.xsd">
+	<name>skywalking_agent</name>
+	<channel>pecl.php.net</channel>
+	<summary>Apache SkyWalking PHP Agent.</summary>
+	<description>The PHP Agent for Apache SkyWalking, which provides the native tracing abilities for PHP project.</description>
+	<lead>
+		<name>jmjoy</name>
+        <user>jmjoy</user>
+        <email>jmjoy@apache.org</email>
+		<active>yes</active>
+	</lead>
```

Comment:
@heyanlong You should add you name in this location.

Reply:
According to ASF policy(ref to Java repo root pom.xml), the author should be Apache SkyWalking, the mail should be dev mail list.

Reply:
I feel it is fine to add your names, but please make sure SkyWalking is listed on the top at least.

Reply:
@jmjoy SkyWalking can be lead, we're developer

Reply:
Like this
```
<lead>
<name>SkyWalking</name>
<email>dev@skywalking.apache.org</email>
</lead>
<lead>
<name>Yanlong He</name>
<user>yanlong</user> <!-- The user block is PECL account ID -->
<email>yanlong@php.net</email>
</lead>

```

Reply:
Alright, but I don't have pecl account.

Reply:
Does the PECL check the user field exists as a PECL account?

Reply:
Changed.